schemas/edges/rxn/rxn_reaction_linked_to_reaction.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
What do you mean by "linked_to"? Maybe you can clarify in the description here.

Reply:
I'm a little leery of being more specific here than the source database but I'll add something

Reply:
Is the linkage some kind of cause and effect, even indirect? Or is it some other relation like similarity?